lib/chromium/root.js

```diff
@@ -180,10 +180,12 @@ var ChromiumTabActor = ActorClass({
   },
 
   form: function(detail) {
+    // Change chrome-specific URLs to something that Firefox can handle.
+    let url = this.json.url.replace(/^chrome:\/\//, "http://chrome-");
```

Comment:
We are trading one invalid URL for another here, precisely for the reason you mention that the URL is only used for crafting a label in the UI.